qiskit_nature/circuit/library/initial_states/_helper.py

```diff
@@ -0,0 +1,197 @@
+# This code is part of Qiskit.
```

Comment:
We usually create a `utils` module in which we place helper functions. See e.g. `qiskit_nature/circuits/library/ansatzes/utils`

Reply:
The situation here seems different because there are no public functions to expose. I don't think it makes sense to create a public `utils` with no public members. Please advise.

Reply:
Simply creating `utils` will not make it part of the public API as long as we do not include it in the sphinx instructions for the documentation.

Reply:
In that case it would be part of the public API, just not part of the documentation. Those are separate things, right? I.e. if I create a `utils` module it you could still import it, it would show up in tab auto-complete. etc.

Reply:
Please just make it consistent with the rest of Qiskit Nature

Reply:
Ok. Just to clarify, you would like me to create a `utils` directory and put `_helper.py` in there. Then, you would like me to add an `__init__.py` to the `utils` directory to initialize it as an empty module.

Reply:
Yes, create a `utils` module with an `__init__` but maybe give the file a more descriptive name than `_helper`. You can also make the file name not start with an underscore at that point

Reply:
Done.